project/Dependencies.scala

```diff
@@ -1,120 +1,117 @@
-  // jaxb-impl 2.3.3 depends on com.sun.activation:jakarta.activation and jakarta.xml.bind:jakarta.xml.bind-api,
-  // which jaxb-impl 2.3.2 did not. jakarta.activation corresponds to the "updated" Maven coordinates for the Java
-  // Activation Framework (https://wiki.eclipse.org/Jakarta_EE_Maven_Coordinates), but Cromwell has many transitive
-  // dependencies on the "old" javax.activation coordinates as well. At assembly time the classes from these two
-  // different jars collide and produce merge conflicts.
-  // It's possible that after updating all other dependencies in Cromwell we might purge the transitive dependencies on
-  // javax.activation and then be able to upgrade to jaxb-impl 2.3.3 or beyond, but some of those other dependencies
-  // such as googleCloudNioV have already been pinned for Scala Steward so this might not be a trivial undertaking.
-  private val jaxbV = "2.3.2" // scala-steward:off
-  private val kindProjectorV = "0.9.9"
-  private val kittensV = "2.2.1"
+  private val hsqldbV = "2.6.0"
+  private val http4sV = "0.21.7" // scala-steward:off (CROM-6678)
+  private val jacksonV = "2.12.5"
+  private val janinoV = "3.1.6"
+  private val jsr305V = "3.0.2"
+  private val kindProjectorV = "0.9.10"
+  private val kittensV = "2.3.2"
   private val liquibaseSlf4jV = "4.0.0"
-  // Scala Steward wanted to upgrade liquibase-core to 3.10.2 but that version does not find some uniqueness
-  // constraints and models datatypes in ways that are incompatible with our test expectations.
-  // liquibase-core 4.0.0 did not have either of those problems but produced tons of strange warnings at runtime
-  // similar in form to this: https://github.com/liquibase/liquibase/issues/1294
-  // Pinning Liquibase version for the time being.
-  private val liquibaseV = "3.6.3" // scala-steward:off
-  private val logbackV = "1.2.3"
-  private val lz4JavaV = "1.7.1"
-  private val mariadbV = "2.7.2"
-  private val metrics3ScalaV = "4.0.0" // https://github.com/erikvanoosten/metrics-scala/tree/f733e26#download-4x
+  private val liquibaseV = "4.4.0" // 4.4.1+ needs https://github.com/liquibase/liquibase/pull/2001
+  private val logbackV = "1.2.5"
+  private val lz4JavaV = "1.8.0"
+  private val mariadbV = "2.7.4"
+  /*
+  The StatsD reporter for DropWizard's (Code Hale's) Metrics 3.x still works with Metrics 4.x.
+  Still would be great to move to Prometheus / OpenCensus
+   */
+  private val metrics4ScalaV = "4.1.19"
   private val metrics3StatsdV = "4.2.0"
   private val mockFtpServerV = "2.8.0"
-  private val mockserverNettyV = "5.5.4" // scala-steward:off (CROM-6582)
-  private val mouseV = "1.0.0"
-  private val mysqlV = "8.0.22" // scala-steward:off (CROM-6772: DateTime/String conversion and connection string updates required)
-  private val nettyV = "4.1.46.Final"
-  private val owlApiV = "5.1.16" // scala-steward:off (CROM-6677)
+  private val mockserverNettyV = "5.11.2"
+  private val mouseV = "1.0.4"
+  private val mysqlV = "8.0.26"
```

Comment:
Some stuff about character sets: https://dev.mysql.com/doc/relnotes/connector-j/8.0/en/news-8-0-26.html#connector-j-8-0-26-feature. Anything to note here?

Reply:
Character encodings, like date-times, are definitely something to keep an eye out for.

In general anyone not using UTC time and 7-bit ASCII may run into issues 😦

That said, we generally hope that any egregious encoding bugs are caught in centaur. For the curious, bugs in the past have popped up when validating and storing values of length "255" in our liquibased tables. What unit is that: bytes, characters, encoded characters in bytes?? If my memory is correct, the issues manifested as strings suddenly being too long to store in some of the tables.